transformer/external/executabletransformer.go

```diff
@@ -32,20 +32,48 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	envMode      = "env"
+	argMode      = "arg"
```

Comment:
Might be good to add a type for different modes, and also have a default.

Reply:
shouldnt the user be forced to specify a input modality. Mode has to be mandatory. No?